nilearn/interfaces/tests/test_bids.py

```diff
@@ -379,25 +379,12 @@ def test_get_bids_files_no_space_entity(tmp_path):
 def test_parse_bids_filename():
     """Check that a typical BIDS file is properly parsed."""
     fields = ["sub", "ses", "task", "lolo"]
-    labels = ["01", "01", "langloc", "lala"]
-    file_name = "sub-01_ses-01_task-langloc_lolo-lala_bold.nii.gz"
+    labels = ["01", "01", "langloc+foo", "lala"]
+    file_name = "sub-01_ses-01_task-langloc+foo_lolo-lala_bold.nii.gz"
```

Comment:
+ are now allowed in entity label, making sure we are OK with this.

Reply:
Fine with me, as long as this is BIDS compliant.

Reply:
that was very recently allowed: we are not in the business of doing BIDS validation, but once in a while it does not hurt to make sure we are not more strict that BIDS is. 😉